client/cli/go/cmds/heketi_storage.go

```diff
@@ -113,9 +118,17 @@ func createHeketiStorageVolume(c *client.Client) (*api.VolumeInfoResponse, error
 	// Create request
 	req := &api.VolumeCreateRequest{}
 	req.Size = HeketiStorageVolumeSize
-	req.Durability.Type = api.DurabilityReplicate
-	req.Durability.Replicate.Replica = 3
 	req.Name = db.HeketiStorageVolumeName
+	req.Durability.Type = dt
+
+	switch dt {
+	case api.DurabilityReplicate:
+		req.Durability.Replicate.Replica = 3
```

Comment:
Do we want to hardcode replica 3 here? or if replica is specified also offer the possibility to specify the count?...

Reply:
This is what there was before. If we want to change it to an other type (arbiter?), that would be a different patch/pr.

Reply:
Ah, yes, I see what you mean. That makes sense.

Do you want that in this patch, or a follow-up one?

Reply:
This one would be good. Simple amend and force-push over, please.